docs/Maintainer-Guidelines.md

```diff
@@ -62,7 +62,7 @@ We now accept versioned formulae as long as they [meet the requirements](Version
 ### Merging, rebasing, cherry-picking
 
 Merging should be done in the `Homebrew/brew` repository to preserve history & GPG commit signing,
-and squash/merge via GitHub should be used for formulae where those formulae
+and squash/merge or rebase/merge via GitHub should be used for formulae where those formulae
```

Comment:
Reading over this again, I feel like this sentence has gotten a bit lengthy at this point. How do you feel about ending the sentence at “GPG commit signing” and starting a new sentence like “Squash/merge or rebase/merge”, etc. Besides that, this would allow us to reword the end of the previous sentence to “...preserve history and GPG commit signing, as the ampersand feels a bit strange to me here.

This is also the one instance where we use “squash/merge” instead of “squash & merge”, so there may be something to be said for standardizing on one style and using it throughout.

These are the only other things that stuck out to me and I’m good with everything else. If you think this is fine without additional changes, then I’m okay to approve it a little later when I have another free moment.

Reply:
Just pushed a new change to improve this. Let me know what you think.